src/main/java/by/mifort/automation/hr/dev/controller/AttributeTypesController.java

```diff
@@ -22,7 +22,7 @@
  */
 
 @RestController
-@RequestMapping("/attributestypes")
+    @RequestMapping("/attributetypes")
```

Comment:
What is the meanning of that space before annotation?

Reply:
fixed, missclick